.github/workflows/compressed.yml

```
@@ -0,0 +1,28 @@
+name: Compressed Size
+
+on:
+  pull_request:
+    branches:
+      - master
+      - canary
```

Hmm, still not showing up. Try this, which is what we have for the other CI actions.

```suggestion
on:
  push:
    branches:
      - master
      - canary
  pull_request:
    branches:
      - master
      - canary
```